From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Thomas Lamprecht <t.lamprecht@proxmox.com>
Date: Tue, 3 Jun 2025 18:19:27 +0200
Subject: [PATCH] d/rules: disable jaeger telemetry

It indirectly causes compile errors when building on Debian Trixie,
but as the jager exporter was dropped from `open-telemetry` 1.10 [0],
it doesn't seem worth to try fixing this.

[0]: https://github.com/open-telemetry/opentelemetry-cpp/pull/2031

Signed-off-by: Thomas Lamprecht <t.lamprecht@proxmox.com>
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 3fbed3f3a2e..9edb8fe1dc4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -21,7 +21,7 @@ ifneq ($(filter pkg.ceph.arrow,$(DEB_BUILD_PROFILES)),)
   extraopts += -DWITH_SYSTEM_ARROW=ON
 endif
 
-extraopts += -DWITH_JAEGER=ON
+extraopts += -DWITH_JAEGER=OFF
 extraopts += -DWITH_SYSTEM_UTF8PROC=ON
 extraopts += -DWITH_OCF=ON -DWITH_LTTNG=ON
 extraopts += -DWITH_MGR_DASHBOARD_FRONTEND=OFF
